packages/components/src/components/FileUploader/FileUploader.stories.tsx

```diff
@@ -0,0 +1,125 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React, { useState } from "react";
+import Uppy from "@uppy/core";
+import { FileUploader } from "./FileUploader";
+import { FileUploaderButton } from "./FileUploaderButton";
+
+export default {
+  component: FileUploader,
+  title: "Components/FileUploader",
+} as ComponentMeta<typeof FileUploader>;
+
+const Template: ComponentStory<typeof FileUploader> = (args) => (
+  <FileUploader {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  label: "Employment contract",
+  maxFileSize: "2MB",
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+export const Loading = Template.bind({});
+Loading.args = {
+  label: "Visa",
+  fileName: "file_to_upload_with_a_long_name_v2.pdf",
+  progress: 50,
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+export const Success = Template.bind({});
```

Comment:
I'm not sure, but shouldn't we render it as mobile here or maybe create a test for it? 
From what I saw, we are not testing mobile designers, right? I also don't know if this is possible with Chromatic.

Reply:
Yes, we can 👯 https://www.chromatic.com/docs/viewports
```
StoryName.parameters = {
  // Set the viewports in Chromatic at a story level.
  chromatic: { viewports: [320, 1200] },
};
```